packages/certs-v5/commands/certs/auto/enable.js

```diff
@@ -11,13 +12,28 @@ async function enable (context, heroku) {
     headers: { 'Accept': 'application/vnd.heroku+json; version=3.cedar-acm' },
     body: {}
   })
-  cli.action.done(`${cli.color.yellow('starting')}. See status with ${cli.color.cmd('heroku certs:auto')} or wait until active with ${cli.color.cmd('heroku certs:auto:wait')}`)
+  if (context.flags.wait) {
+    cli.action.done(`${cli.color.yellow('starting')}.`)
+  } else {
+    cli.action.done(`${cli.color.yellow('starting')}. See status with ${cli.color.cmd('heroku certs:auto')} or wait until active with ${cli.color.cmd('heroku certs:auto:wait')}`)
```

Comment:
Should this be `certs:auto --wait` ? I don't know why the old output showed `heroku certs:auto:wait` since there doesn't seem to be a command that matches that.

Reply:
I wasn't sure about that so I left it unchanged, It looks like someone tried [to implement that a few years ago](https://github.com/heroku/cli/commit/459846e2) but left it unimplemented.

I can change the comment to heroku certs:auto:enable --wait, but I don't know if it's ok to call the enable command twice, or I can add the certs:auto:wait command.

Reply:
Hmm. That or we can use the same notify functionality in both enable and index?

Either way, seems fine to call enable repeatedly:
```
% heroku certs:auto:enable -a matt-acm-app
Enabling Automatic Certificate Management... starting. See status with heroku certs:auto or wait until active with heroku certs:auto:wait
=== Your certificate will now be managed by Heroku.  Check the status by running heroku certs:auto.

% heroku certs:auto:enable -a matt-acm-app
Enabling Automatic Certificate Management... starting. See status with heroku certs:auto or wait until active with heroku certs:auto:wait
=== Your certificate will now be managed by Heroku.  Check the status by running heroku certs:auto.
```

Reply:
🤔  Using the wait flag on index looks weird to me and I don't see other commands using that pattern

Reply:
I talked about this with the pillow 😅  and it convinced me to add the flag to certs:auto